lib/storage/gcs.js

```diff
@@ -0,0 +1,152 @@
+/* jshint node:true */
+
+// Google Cloud Storage backend for uploadfs. See also
+// local.js.
+
+var {Storage} = require('@google-cloud/storage');
```

Comment:
Please stick to idioms supported back to Node 6. `const` and `let` are fine but object destructuring is not supported.

Reply:
i.e. `var Storage = require('@google-cloud/storage').Storage;`